docs/csharp/tutorials/string-interpolation.md

```diff
@@ -1,122 +1,98 @@
+## How to use escape sequences in an interpolated string
 
-Which would output:
+Interpolated strings support all escape sequences that can be used in ordinary string literals. For more information, see [String escape sequences](../programming-guide/strings/index.md#string-escape-sequences). To include a brace, "{" or "}", in the result string, use two braces, "{{" or "}}". For more information, see the [Escaping Braces](../../standard/base-types/composite-formatting.md#escaping-braces) section of the [Composite Formatting](../../standard/base-types/composite-formatting.md) topic.
 
-```
-This is line number 1
-This is line number 2
-This is line number 3
-This is line number 4
-This is line number 5
-```
+[Verbatim](../language-reference/tokens/verbatim.md) interpolated strings start with the `$` character followed by the `@` character.
 
-## How string interpolation works
+The following example shows how to include braces into the result string and construct a verbatim interpolated string:
 
-Behind the scenes, this string interpolation syntax is translated into `String.Format` by the compiler. So, you can do the [same type of stuff you've done before with `String.Format`](../../standard/base-types/formatting-types.md).
+[!code-csharp-interactive[escape sequence example](~/samples/snippets/csharp/tutorials/string-interpolation/Program.cs#4)]
 
-For instance, you can add padding and numeric formatting:
+## How to use a ternary conditional operator in an interpolated expression
 
-[!code-csharp[Interpolation formatting example](../../../samples/snippets/csharp/new-in-6/string-interpolation.cs#InterpolationFormattingExample)]  
+As the colon (":") has special meaning in an item with an interpolated expression, in order to use a [conditional operator](../language-reference/operators/conditional-operator.md) in an expression, enclose it in parentheses, as the following example shows:
 
-The above would output something like:
+[!code-csharp-interactive[conditional operator example](~/samples/snippets/csharp/tutorials/string-interpolation/Program.cs#5)]
 
-```
-998        5,177.67
-999        6,719.30
-1000       9,910.61
-1001       529.34
-1002       1,349.86
-1003       2,660.82
-1004       6,227.77
-```
+## How to create a culture-specific result string with string interpolation
 
-If a variable name is not found, then a compile-time error is generated.
+By default, an interpolated string resolves to the result string with the use of the current culture defined by the <xref:System.Globalization.CultureInfo.CurrentCulture> property. Use implicit conversion of an interpolated string to a <xref:System.FormattableString?displayProperty=nameWithType> instance and its <xref:System.FormattableString.ToString(System.IFormatProvider)> method to create a culture-specific result string. The following example shows how to do that:
```

Comment:
- The first sentence is a bit awkward. Perhaps, "an interpolated string uses the current culture defined by the \<xref:System.Globalization.CultureInfo.CurrentCulture?displayProperty=nameWithType> property for all formatting operations." 
- "and its" --> and call its